core/messages/common/en.ts

```diff
@@ -4,13 +4,13 @@ export const commonMessages = i18n('common', {
   brokenCategory: 'Broken category',
   empty: 'The value is required',
   generalCategory: 'General',
-  internalError: 'The app crashed, try again later when we’ll fix it',
-  invalidCredentials: 'There is no user with such user ID and password',
-  invalidSecret: 'Secret should have 2 words by 10 symbols',
-  invalidServer: 'Is not valid domain',
-  invalidUrl: 'Doesn’t look like web page address',
-  invalidUserId: 'User ID should contain 16 numbers',
-  loading: 'Loading',
-  networkError: 'Can’t reach server, check your network',
+  internalError: 'Something went wrong. Please try again later while we fix it.',
```

Comment:
```suggestion
  internalError: 'The app crashed. Please try again later while we fix it.',
```

`Something went wrong` is very popular wording in this case. I want to avoid feeling of big tech, so prefer to be more honest.